README.md

```diff
@@ -1,2 +1,21 @@
 # hw01
 Repository for homework 01
+
+#**Sarah Jensen**
```

Comment:
# Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |           |      x        |            |
| **Achievement, creativity** |           |      x        |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* You failed to put a space after # to create a header multiple times--a mistake it appears many others in our course (including myself), made as well.
* The nature of a hyperlink makes it so you can place the hyperlink in one of the first words of your introduction to the linked page. There is no need to introduce the page and then use the same words for a separate hyperlink--meaning you could have consolidated your lines and made the header "Bio and Photo" a hyperlink itself.
* You did great with the list and embedding the photo of the pizza!
* I think a big takeaway from this assignment is that we all need to pay more attention to the minutiae of our code, like spaces.
* I am direct copying this from my review of another peer's assignment because I feel the message applies to you (and I) as well and is important: I think many people in the class sought to complete everything the assignment asked of us, but did not go beyond the specific instructions. Programming is not a discipline that people typically think of as "creative" but we need to challenge that perspective in this course and strive to be creative and even innovative with our work.

Reply:
Required a space after #